docs/user_guide/service_config.md

```diff
@@ -977,6 +977,37 @@ Example:
 admissionController.accessControl.externalGroups: "^sales$,^marketing$,^admin-"
 ```
 
+### Compresion setting
+
+The data in ConfigMap cannot exceed 1 MiB. YuniKorn supports Gzip algorithm to decompress data in the `binaryData` field.
+If a key ends with `.gz`, YuniKorn decodes the value by base64 and decompress it by Gzip algorithm.
```

Comment:
How about: "If a key ends with `.gz`. YuniKorn will treat the value as gzip-compressed data and decompress it automatically. The base64 encoding is automatically "